shell/components/__tests__/ContainerResourceLimit.test.ts

```diff
@@ -0,0 +1,35 @@
+import { mount } from '@vue/test-utils';
+import ContainerResourceLimit from '@shell/components/ContainerResourceLimit.vue';
+
+describe('component: ContainerResourceLimit', () => {
+  it.skip.each([
+    ['limitsCpu', 'cpu-reservation', '111m', '111'],
+    ['limitsMemory', 'memory-reservation', '111Mi', '111'],
+    ['requestsCpu', 'cpu-limit', '111m', '111'],
+    ['requestsMemory', 'memory-limit', '111Mi', '111'],
```

Comment:
It looks like the ids aren't matching. I think that we can remove the skip if we make this change

```suggestion
    ['limitsCpu', 'cpu-limit', '111m', '111'],
    ['limitsMemory', 'memory-limit', '111Mi', '111'],
    ['requestsCpu', 'cpu-reservation', '111m', '111'],
    ['requestsMemory', 'memory-reservation', '111Mi', '111'],
```

Reply:
Oh that makes sense 😅 
Thanks for noticing